resources/keywords/deploy_instance_dev.resource

```diff
@@ -38,7 +38,7 @@ Set Project Namespace To ${proj}
     Click Element    ${btnProjectSelectUpd}
     Element Should Be Visible    ${lblProjectSelectUpd}
 
-User Navigates To Connect ${databaseProvider} Screen On Developers View
+User Navigates Add ${databaseProvider} To Topology Screen
```

Comment:
Will it be possible to add change the keyword to - User Navigates **To** Add ${databaseProvider} To Topology Screen. Please ignore if too trivial